lib/webview-helpers.js

```diff
@@ -382,13 +382,8 @@ helpers.getWebViewsMapping = async function getWebViewsMapping (adb, {
   androidDeviceSocket = null,
   ensureWebviewsHavePages = true,
   webviewDevtoolsPort = null,
-  enableWebviewDetailsCollection = true,
-  isChromeSession = false
+  enableWebviewDetailsCollection = true
```

Comment:
Please update the corresponding docstring

Reply:
I missed 3a34fc08ba4c4d2109c808170f4065f9013c269a ...